components/python/python27/test/results-32.master

```diff
@@ -0,0 +1,29 @@
+373 tests OK.
```

Comment:
Why only 32-bit tests results are there?

Reply:
The reason is now in the PR descriptions, was "hiding" in on of the commit descriptions. Looked as if all possible file descriptors were taken... Would be cool if someone could provide that 64-bit results file.